File: CuponSmart/src/cuponsmart/modelo/pojo/respuesta/RespuestaEstatus.java
package cuponsmart.modelo.pojo.respuesta;

import java.util.List;
import cuponsmart.modelo.pojo.entidad.Estatus;

public class RespuestaEstatus extends Mensaje{
    private List<Estatus> contenido;
    
    public List<Estatus> getContenido(){
        return contenido;
    }
    
    public void setContenido(List<Estatus> contenido){
        this.contenido = contenido;
    }
}

File: AppCuponSmart/app/src/main/java/uv/tc/appcuponsmart/data/model/respuesta/RespuestaEstatus.kt
package uv.tc.appcuponsmart.data.model.respuesta

import uv.tc.appcuponsmart.data.model.entidad.Estatus

class RespuestaEstatus(
    error: Boolean,
    mensaje: String,
    var contenido: MutableList<Estatus>? = null
): Mensaje(error, mensaje)